docs/source/tiles.rst

```diff
@@ -3,9 +3,6 @@ Tiles
 
 .. note::
   The tiles endpoint currently match only partially (the GET endpoints) the default Plone implementation.
-  The serialization of tiles didn't match the Mosaic (and plone.app.blocks) implementation and it's done to
```

Comment:
@sneridagh shouldn't we leave a note here that at some point we are going to remove that endpoint? As far as we know nobody ever used that endpoint anyways.